### ISSUE TITLE
Relative time on admin page is wrong

### ISSUE THREAD

**Issue by jvmdc** [juice-shop/multi-juicer#204]
It seems the logic surrounding the relative times for the "Created" and "Last used" column is wrong.
![image](https://github.com/juice-shop/multi-juicer/assets/109624469/0ce7de3a-da2d-45f2-aad3-8c94e756df9e)
In the screenshot above taken today (2024-03-05), notice that it says "1 month ago". Hovering the mouse over the relative time reveals that it was in fact only on 2024-02-29, i.e. less than a week ago

**Comment by bkimminich** [comment]
Maybe it just means "last month", which would be correct looking at the February date from March?

**Comment by jvmdc** [comment]
Yup, I would agree - the wording should probably say "Last month" instead then though.

I'd argue that counting days is better since "Last month" leaves me not knowing at a glance whether it was 6 days ago or 30 days ago.

**Comment by jvmdc** [comment]
Without having taken a deep-dive in the code, I think the issue comes from the use of "selectUnit". Notice the "Caveats" listed here: https://www.npmjs.com/package/@formatjs/intl-utils#caveats (Also notice that the package is deprecated and selectUnit has been removed from the updated package)

**Comment by J12934** [comment]
Admin page, including the timestamps have been reworked in 8.0.0, and should now reflect the proper time :)
